srcpkgs/cargo-update/template

```diff
@@ -10,7 +10,7 @@ maintainer="Jan Christian Grünhage <jan.christian@gruenhage.xyz>"
 license="MIT"
 homepage="https://github.com/nabijaczleweli/cargo-update"
```

Comment:
they don't write a changelog to a text file, but the releases page contains the release notes
```suggestion
homepage="https://github.com/nabijaczleweli/cargo-update"
changelog="https://github.com/nabijaczleweli/cargo-update/releases"
```